### ISSUE TITLE
PLI COPYBOOK parsing issue

### ISSUE THREAD

**Issue by kraabrsg** [IBM/zopeneditor-about#211]
## Development environment used

- Z Open Editor version:
- Editor Platform
  - [x] Visual Studio Code
  - [ ] Red Hat CodeReady Workspaces
  - [ ] Eclipse Che
  - [ ] Standalone Theia
- Editor Platform Version:
- Operating System (on which VS Code runs such as Windows 10 2004, otherwise name and version of platform such as OpenShift v4.3): WIN 10 Enterporse 10.0.19042 Build 19042
- Java Version (when using VS Code or Theia, execute `java -version` and paste the details here): 
- "zopeneditor.JAVA_HOME": "C:\\xxxxxxx\\java\\jdk-11.0.2",
- [ ] Related to RSE API?
  - RSE API Plugin version:
  - Zowe CLI version: 6.37.0
  - Node.js version: 14.15.3
- Logs attached (see [here](https://ibm.github.io/zopeneditor-about/Docs/locating_local_client_logs.html) how to get them): yes

## Problem Description

Detailed steps for reproducing the problem:
Taking this dummy source:

```
 XNAMEXX: PROC OPTIONS(MAIN);
 
 dcl Infile file record input;
 
 DCL InfileEnv  char(4);
 InfileEnv = FILEDDINT(Infile,'RECSIZE');              /* xxxxx */

 DCL InFileRecl DEC FIXED(3) INIT(123);
 
 %include KTEST2;
 
 DCL TESTRESULT BIT('0'B);

 OPEN FILE(INFILE);
 CLOSE FILE(INFILE);

 TESTRESULT = TESTPROC('Y','1'B); 

 TESTPROC: PROC(YY,XX) RETURNS(BIT(1));
    DCL YY CHAR(1);
    DCL XX BIT(1);
    RETURN(XX);
 END TESTPROC;

 
 END XNAMEXX;
```

1. First step

### Observed behavior
Yields to:
![grafik](https://user-images.githubusercontent.com/86772851/148543671-ca6d5b00-36dc-42f1-a958-6a0852123629.png)

### Expected behavior
Formating should be shown like in the host file...

Copybook:
[ktest2.txt](https://github.com/IBM/zopeneditor-about/files/7828777/ktest2.txt)

In the copybook all pre processor variables are removed. They are mainly used to differ the needs for "caller" and "called", without the pre processor statements this seems confusing. But the parsing does not work here like expected.

Thanks!

**Comment by FALLAI-Denis** [comment]
"packed" pds member ?

[Packed data sets](https://www.ibm.com/docs/en/zos/2.1.0?topic=sets-packed-data)

**Comment by kraabrsg** [comment]
> 

Hi @FALLAI-Denis both no, but thank you for taking you time looking into that!

**Comment by FALLAI-Denis** [comment]
FB or VB pds?

**Comment by kraabrsg** [comment]
FB 80, PO

**Comment by phaumer** [comment]
If the files are very long we cut them off to not slow down the UI, but when that happens they should still render correctly, which we need to investigate.

**Comment by phaumer** [comment]
Fixed in Z Open Editor v2.0.0
